src/url.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+
+const URLS = {
+  home: () =>
+    "/",
+  collectionRecords: ({bid, cid}) =>
+    `/buckets/${bid}/collections/${cid}/records`,
+  recordAttributes: ({bid, cid, rid}) =>
+    `/buckets/${bid}/collections/${cid}/records/${rid}/attributes`,
```

Comment:
You mean like in Django? Not any that I'm aware of, but if you find one I take it :)